source/FreeRTOS_TCP_Reception.c

```diff
@@ -369,9 +344,9 @@
  * @param[in] uxIndex: Index of options in the TCP packet options.
  * @param[in] pxSocket: Socket handling the TCP connection.
  */
-        _static void prvReadSackOption( const uint8_t * const pucPtr,
-                                        size_t uxIndex,
-                                        FreeRTOS_Socket_t * const pxSocket )
+        void prvReadSackOption( const uint8_t * const pucPtr,
```

Comment:
Why did we remove the static qualifier? I think this function is local to this file.

I think we should keep the static qualifier and remove the declaration from the header.

Reply:
These two were added to the header file for running unittest.